doc/configuration.rst

```diff
@@ -99,6 +99,33 @@ Subclassing ``Config`` or ``DefaultConfig``
 
     Defaults to ``False``.
 
+  .. method:: cli_context(command: str) -> Iterator[None]
+
+    A context manager which wraps the execution of the CLI command. Use it to
+    perform any initialization or cleanup required by your codebase.
+
+    ``command`` is the command passed to the monkeytype cli: ``'run'``,
+    ``'apply'``, etc.
+
+    Since :meth:`cli_context` is used as a context manager, any :class:`Config`
+    subclass that implements this method needs to decorate it with a context
+    decorator like :meth:`contextlib.contextmanager`, and needs to yield in
+    the body of the method exactly once. Code that comes before the yield will
+    run before the command starts, and code that comes afterward will run when
+    the command finishes but before the CLI exits.
+
+    For example, if you run MonkeyType against a Django codebase, you can use
+    this method to setup Django before the command runs::
+
+      @contextmanager()
+      def cli_context(self, command: str) -> Iterator[None]:
+          import django
+          django.setup()
+          yield
+
+    The default implementation of this method just calls ``yield`` to let the
```

Comment:
"The default implementation of this method just returns a no-op context manager."